src/common/components/giftViews/giftListItem/giftListItem.component.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
+import template from './giftListItem.tpl';
+import desigSrc from 'common/directives/desigSrc.directive';
+
+let componentName = 'giftListItem';
+
+class GiftListItemController {
+
+  /* @ngInject */
+  constructor() {
+  }
+}
+
+export default angular
+  .module( componentName, [
+    template.name,
+    desigSrc.name
+  ] )
+  .component( componentName, {
+    controller:  GiftListItemController,
+    templateUrl: template.name,
+    transclude:  {
+      'selectInput': '?label'
+    },
+    bindings:    {
+      gift:        '<',
```

Comment:
Is https://github.com/CruGlobal/give-web/pull/190/files#diff-119da8a34d16697efc4a9919ad4bc57eR49 updating correctly? Seems like it might not always run that when gift is updated since it isn't watching it. Idk